packages/registry/contracts/CannonRegistry.sol

```diff
@@ -3,10 +3,13 @@ pragma solidity ^0.8.11;
 
 contract CannonRegistry {
   error Unauthorized();
-  error InvalidUrl();
+  error InvalidUrl(string url);
+  error InvalidName(bytes32 name);
```

Comment:
This is nitpick, but don't think the addition of url & name is necessary, as for the user is clear to which url the `InvalidUrl` error is, same with name. And makes the contract simplier.

Reply:
lets leave it be for now